lib/dl_file_uploader_lib/dl_file_uploader_lib/redis_model/models/models.py

```diff
@@ -195,10 +196,14 @@ class DataFile(RedisModelUserIdAuth):
     error: Optional[FileProcessingError] = attr.ib(default=None)
 
     KEY_PREFIX: ClassVar[str] = "df"
-    DEFAULT_TTL: ClassVar[Optional[int]] = 12 * 60 * 60  # 12 hours
+    DEFAULT_TTL: ClassVar[Optional[int]] = 3 * 60 * 60  # 3 hours
 
     @property
-    def s3_key(self) -> str:
```

Comment:
nitpick: sound's like it was easier to make `s3_key_new` field and rename it later to `s3_key`. would be way less code changes